typings/index.d.ts

```diff
@@ -949,7 +949,7 @@ declare module 'discord.js' {
 		public attachments: Collection<Snowflake, MessageAttachment>;
 		public author: User;
 		public channel: TextChannel | DMChannel;
-		public readonly cleanContent: string;
+		public readonly cleanContent: string | null;
```

Comment:
This should only be `null` for partial messages, right?
In that case `Partialize<T>` would already take care of that.
```suggestion
		public readonly cleanContent: string;
```